plugin/evm/warp/backend.go

```diff
@@ -11,11 +11,16 @@ import (
 	"github.com/ava-labs/avalanchego/database"
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
 	"github.com/ava-labs/avalanchego/utils/hashing"
 	"github.com/ava-labs/avalanchego/vms/platformvm/teleporter"
 )
 
-var _ WarpBackend = &warpBackend{}
+var (
+	_ WarpBackend = &warpBackend{}
+
+	emptySignature = [bls.SignatureLen]byte{}
```

Comment:
I assume emptySignature is an invalid signature and can never be mistaken for a valid one. 

(For my own understanding): Is it easier/cleaner for the caller to handle an empty invalid signature instead of a nil?

Reply:
Can we remove this to make the code more obviously correct?

This will only be returned when there is a non-nil error.

Reply:
re @bernard-avalabs , not sure if it's cleaner or necessarily easier to handle the empty signature case, but more so changed because we want `warpBackend.GetSignature` to return the expected signature type, and with that we can't return nil as type `[bls.SignatureLen]byte`